hatchery/hatchery.go

```diff
@@ -326,13 +351,22 @@ func createECSCluster(w http.ResponseWriter, r *http.Request) {
 	var reader *strings.Reader
 	if err != nil {
 		reader = strings.NewReader(err.Error())
-		Config.Logger.Printf("Error: %s", err)
+		Config.Logger.Errorw("Error creating ECS cluster",
+			"error", err.Error(),
+			"username", userName,
+			"paymodel", payModel,
+		)
 	} else {
 		reader = strings.NewReader(result.String())
 	}
+	// TODO: What's happening here?
```

Comment:
should we add more details to the todo or is it a reminder for yourself that we can remove? 😄

Reply:
It was a reminder for myself 😄  I still don't know what's happening, but I will remove this comment.